### DISCUSSION TITLE
Publish PWA app stores

### DISCUSSION THREAD

**Discussion by stokito** [super-productivity/super-productivity#3195] [General] [answered]
https://store.app/
https://pwa.ist/ (not sure if you can add yourself)
This may attract more users.

**Comment by johannesjo** [comment, marked as answer]
Thank you very much for the suggestion!! <3 Will do this!

**Reply by johannesjo** [reply]
Good reviews are probably helpful :D

https://store.app/app-super-productivity-com?share=6b0eb4768cee1807bdddb571503986a7&utm_medium=organic_social&utm_campaign=user_share

**Reply by stokito** [reply]
Thank you!
